xbmc/guilib/GUIFontTTF.cpp

```diff
@@ -732,7 +732,7 @@ bool CGUIFontTTFBase::CacheCharacter(wchar_t letter, uint32_t style, Character *
       m_posX += -bitGlyph->left;
 
     // check we have enough room for the character
-    if ((m_posX + bitGlyph->left + bitmap.width) > static_cast<int>(m_textureWidth))
+    if ((m_posX + bitGlyph->left + bitmap.width) > m_textureWidth)
```

Comment:
hmm. then there are platform / version  / compiler differences. this definitely quells the warning on my machine, the lhs is unsigned here.
with cast:
```
/home/akva/kode/xbmc-cmake/xbmc/guilib/GUIFontTTF.cpp: In member function ‘bool CGUIFontTTFBase::CacheCharacter(wchar_t, uint32_t, CGUIFontTTFBase::Character*)’:
/home/akva/kode/xbmc-cmake/xbmc/guilib/GUIFontTTF.cpp:735:50: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
     if ((m_posX + bitGlyph->left + bitmap.width) > static_cast<int>(m_textureWidth))
```
without cast it's silent. fun! ubuntu 16.04 / gcc 5.4 / libfreetype6-dev=2.6.1-0.1ubuntu2